phpseclib/Math/BinaryField/Integer.php

```diff
@@ -160,7 +155,7 @@ private static function deg($x)
      * @return string[]
      * @link https://en.wikipedia.org/wiki/Polynomial_greatest_common_divisor#Euclidean_division
      */
-    private static function polynomialDivide($x, $y)
+    private static function polynomialDivide($x, $y): array
```

Comment:
Should be able to do `string $x, string $y`

Reply:
Done